spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/TaskRepository.java

```diff
@@ -68,4 +68,24 @@ TaskExecution completeTaskExecution(long executionId, Integer exitCode, Date end
 	TaskExecution createTaskExecution(String taskName,
 			Date startTime,List<String> arguments);
 
+	/**
+	 * Notifies the repository that an empty taskExecution needs to be created. TaskName, startTime are set to null
```

Comment:
Can we change this to something like "Creates an empty `TaskExecution` with just an id provided.  This is intended to be utilized in systems where the request of launching a task is separate from the actual start of a task (the underlying system may need to deploy the task prior to launching, etc)."?